.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
+bower_components
+*.swp
```

Comment:
I lean toward only putting in gitignore what is created by tools in the repo itself. "bower_components" makes sense here, but `core-meta` doesn't use NPM so node_modules doesn't quite belong either.

Just a preference.